src/app/frontend/resource/workloads/pod/detail/template.html

```diff
@@ -80,14 +80,24 @@
       <div value>{{pod.restartCount}}</div>
     </kd-property>
 
+    <kd-property *ngIf="pod?.serviceAccountName !== undefined">
```

Comment:
As this is a string then `ngIf="pod?.serviceAccountName` is sufficient here. It will only be true for non-empty strings.

Reply:
Another lesson learned. Thank you.